plugin/trino-exchange/src/main/java/io/trino/plugin/exchange/s3/ExchangeS3Config.java

```diff
@@ -176,4 +180,29 @@ public ExchangeS3Config setAsyncClientConcurrency(int asyncClientConcurrency)
         this.asyncClientConcurrency = asyncClientConcurrency;
         return this;
     }
+
+    @Min(1)
+    public int getAsyncClientMaxPendingConnectionAcquires()
+    {
+        return asyncClientMaxPendingConnectionAcquires;
+    }
+
+    @Config("exchange.s3.async-client-max-pending-connection-acquires")
```

Comment:
@arhimondr @losipiuk .. did we not want to document this property?

Reply:
It's pretty low level and it should generally not be changed. Not sure if it makes sense to document.